features/native_crash_handling.feature

```diff
@@ -190,3 +190,22 @@ Feature: Native crash reporting
         And the event "severity" equals "error"
         And the event "unhandled" is true
 
+    Scenario: Causing a crash in a separate library
```

Comment:
This scenario fails on an x86, API 27 emulator, with the following error message:

>And the "method" of stack frame 0 equals "something_innocuous"                                                                 # maze-runner-24f53756011b/lib/features/steps/error_reporting_steps.rb:103
      Expected: "something_innocuous"
        Actual: "bsg_unwind_stack_libunwind" (Minitest::Assertion)
      features/native_crash_handling.feature:208:in `And the "method" of stack frame 0 equals "something_innocuous"'